packages/nc-gui/components/cell/attachment/index.vue

```diff
@@ -150,7 +150,12 @@ watch(
 useSelectedCellKeyupListener(inject(ActiveCellInj, ref(false)), (e) => {
   if (e.key === 'Enter' && !isReadonly.value) {
     e.stopPropagation()
-    modalVisible.value = true
+    if (!modalVisible.value) {
```

Comment:
It will be better to open the file picker on enter if there is no attachments? We can pick it up later if actually needed

Reply:
May follow up this later.